workflow-dev/tpl/generate_params.toml

```diff
@@ -47,3 +49,58 @@ key_json = '''Paste JSON data here.'''
 registry_bucket = "your-regsitry-bucket-name"
```

Comment:
Fixed in https://github.com/deis/charts/pull/228/commits/674224f2f7e54b7ab695f014a63924d1ac673f16